src/governance/classes.cpp

```diff
@@ -25,8 +23,7 @@ CGovernanceTriggerManager triggerman;
 // http://www.boost.org/doc/libs/1_58_0/doc/html/boost/algorithm/split_idp202406848.html
 std::vector<std::string> SplitBy(const std::string& strCommand, const std::string& strDelimit)
 {
-    std::vector<std::string> vParts;
-    boost::split(vParts, strCommand, boost::is_any_of(strDelimit));
+    std::vector<std::string> vParts = SplitString(strCommand, strDelimit);
```

Comment:
25057: nit: should drop 
```
// http://www.boost.org/doc/libs/1_58_0/doc/html/boost/algorithm/split_idp202406848.html
```
above.

Also, I'm wondering what is actually going on here and why do we need the loop below (and why we couldn't simply use `boost::split`/`SplitString` instead of `SplitBy`) 🤔 but that's out of the scope for this PR :)

Reply:
Resolved with latest push